src/lib/components/datetime/RelativeDateTime.tsx

```diff
@@ -1,9 +1,27 @@
+import type {FormatDistanceToken} from 'date-fns';
 import {formatDistanceToNowStrict} from 'date-fns/formatDistanceToNowStrict';
 
-export default function RelativeDateTime({date}: {date: Date}) {
+export default function RelativeDateTime({date, suffix}: {date: Date; suffix: string}) {
   return (
     <time dateTime={date.toISOString()}>
-      {formatDistanceToNowStrict(date, {addSuffix: true, roundingMethod: 'floor'})}
+      {formatDistanceToNowStrict(date, {
+        addSuffix: true,
+        roundingMethod: 'floor',
+        locale: {
+          formatDistance: (token: FormatDistanceToken, count: number) => {
+            const formatMap: Partial<Record<FormatDistanceToken, string>> = {
+              xSeconds: `${count}s ${suffix}`,
+              xMinutes: `${count}min ${suffix}`,
+              xHours: `${count}hr ${suffix}`,
+              xDays: `${count}d ${suffix}`,
+              xWeeks: `${count}w ${suffix}`,
+              xMonths: `${count}mo ${suffix}`,
+              xYears: `${count}y ${suffix}`,
```

Comment:
What is the `x` prefix for?

Reply:
The `xSeconds` is the locale listed here: https://github.com/date-fns/date-fns/blob/main/src/locale/en-US/_lib/formatDistance/index.ts. By default, xSeconds => {count} seconds, but the full word takes up too much space in the toolbar, so we need to customize it to use the short form